tests/validate/43-jinja2-template-error-main.t

```diff
@@ -33,7 +33,7 @@ __SUITERC__
 run_fail "${TEST_NAME_BASE}" cylc validate 'suite.rc'
 contains_ok "${TEST_NAME_BASE}.stderr" <<'__ERROR__'
 Jinja2Error:
-    'You can only sort by either "key" or "value"'
+    raise FilterArgumentError('You can only sort by either "key" or "value"')
```

Comment:
The other changes are the result of copy-and-paste of Jinja2 2.11.1 ZIP contents, minus their asyncio files as that breaks the build due to Py3 code.

After copying the files, this was the only test that failed. Debugging it, it looks like the traceback contents changed in 2.11.1, and instead of having just the text on a new line, it contains the `raise FilterArgumentError` part too.

Wondering if that's safe to update... I don't think users rely on this?

Reply:
Seems fine to me.

Reply:
(What were the asyncio files for though?   Is it surprising that deleting those didn't entirely break the new Jinja2?)

Reply:
Curious, 2.11.1 claims to have 2.7 support. What aspect of the build was breaking? Was is our tests (e.g. pycodestyle) or something functional?